### PR TITLE
Add update instructions for RPM/DEB and bin distribution

### PR DESCRIPTION
**Pull request by tbouron**
This requires the `paths.html` page available with #210

### CODE REVIEW

**Comment by tbouron** [comment]
I added the links for the DEB package on the download pages. This is subject to our ability of building the DEB and RPM package for snapshot though /cc @richardcloudsoft

File: guide/ops/upgrade.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+          # CentOS 7 / RHEL
+          sudo systemctl start brooklyn
+          # CentOS 6 and older
+          sudo initctl start brooklyn
+          
+          # Ubuntu / Debian
+          start brooklyn
+
+   3. Check whether you have rebind ERROR messages in `./log/brooklyn.info.log`, e.g. `sudo grep -E "WARN|ERROR" /opt/brooklyn/log/brooklyn.debug.log`.
```
Comment (aledsage) [inline guide/ops/upgrade.md]: I'd say "ERROR mesages in the brooklyn log, e.g. ..." (rather than the `./log/...`)

Comment (aledsage) [reply guide/ops/upgrade.md]: I'd use `sudo grep -E "Rebind|WARN|ERROR" /var/log/brooklyn/brooklyn.info.log` (important to include something like "Rebind" so the user sees a message about success, as well as some benign (?) warnings).

File: guide/ops/upgrade.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+          sudo yum erase apache-brooklyn
+          
+          # Ubuntu / Debian
+          sudo dpkg -r apache-brooklyn
+
+    2. On CentOS 7 run `sudo systemctl daemon-reload`.
+
+    3. Confirm that Brooklyn is definitely not running (see step 1 above).
+
+    4. Delete the Brooklyn install directory: `sudo rm -r /opt/brooklyn`.
```
[the remark this answers is not in the document]
Comment (tbouron) [reply guide/ops/upgrade.md]: Good catch, will amend

File: guide/ops/upgrade.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+   
+   2. Install Apache Brooklyn:
+
+          # CentOS / RHEL
+          sudo yum install apache-brooklyn-xxxx.noarch.rpm
+          
+          # Ubuntu / Debian
+          sudo dpkg -i apache-brooklyn-xxxx.all.deb
+
+   3. Wait for Brooklyn to be fully started (i.e. its web-console to be responsive).
```
[the remark this answers is not in the document]
Comment (tbouron) [reply guide/ops/upgrade.md]: Brooklyn does not start automatically, I'm missing a step here. Thanks @aledsage

File: guide/ops/upgrade.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+
+      Confirm that Brooklyn is no longer running (see step 1).
+
+   2. Restore the persisted state directory into `/var/lib/brooklyn` and any changes to the configuration files (see step 2).
+      Ensure owner/permissions are correct for the persisted state directory, e.g.:
+      `sudo chown -r brooklyn:brooklyn /var/lib/brooklyn`
+
+7. Validate that the new release works, by starting in "HOT_BACKUP" mode.
+
+   1. Before starting Brooklyn, reconfigure `./etc/org.apache.brooklyn.osgilauncher.cfg` and set `highAvailabilityMode=HOT_BACKUP`.
```
[the remark this answers is not in the document]
Comment (tbouron) [reply guide/ops/upgrade.md]: Indeed, will amend

File: guide/ops/upgrade.md (hunk 1 above)
Comment (tbouron) [reply guide/ops/upgrade.md]: Make sense @aledsage, will update

**Comment by tbouron** [comment]
@aledsage Comments addressed

**Pull request merged**